### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by pontus**

### CODE REVIEW

File: charts/grand-challenge/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# A chart can be either an 'application' or a 'library' chart.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+
+version: 0.1.0
+
+# There are many parts of the cytomine system, so the cytomine-core version is
```
Comment (shreyasshivakumara) [inline charts/grand-challenge/Chart.yaml]: Grand-challenge here?

File: charts/grand-challenge/templates/celery-beat-networkpolicy.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - Ingress
+  ingress:
+  egress:
+    - to:
+        - podSelector:
+            matchLabels:
+              service: postgres
+      ports:
+        - protocol: TCP
+          port: 5432
```
Comment (shreyasshivakumara) [inline charts/grand-challenge/templates/celery-beat-networkpolicy.yaml:20]: Any reason not to template the network policies?

Comment (pontus) [reply charts/grand-challenge/templates/celery-beat-networkpolicy.yaml:20]: I may be missing something that should be templated, but in general, I try to template what's needed and not template what's no. Is there anything it seems I've missed here that you can spot?

**Review by shreyasshivakumara — APPROVED** [review]
Looks good! :+1:

File: charts/grand-challenge/Chart.yaml (hunk 1 above)
Comment (pontus) [reply charts/grand-challenge/Chart.yaml]: Thanks, fixed.

File: charts/grand-challenge/templates/celery-beat-networkpolicy.yaml (hunk 2 above)
Comment (shreyasshivakumara) [reply charts/grand-challenge/templates/celery-beat-networkpolicy.yaml:20]: I see you have a different style compared to how the Cytomine helm chart is written. I don't spot anything but I was curious to know if there was any specific reason why didn't template the network policy ports and other stuff.

**Comment by shreyasshivakumara** [comment]
Can I merge it now?

File: charts/grand-challenge/templates/celery-beat-networkpolicy.yaml (hunk 2 above)
Comment (pontus) [reply charts/grand-challenge/templates/celery-beat-networkpolicy.yaml:20]: Ah. Yes, to me, it's easier to read and work with if things aren't templated (unless it's needed/brings value).

**Pull request merged**